src/main/java/control/TicketService.java

```diff
@@ -0,0 +1,103 @@
+package main.java.control;
```

Comment:
set correct packages names for Maven. For example, `package control;` instead of `package main.java.control;`

Reply:
Fixed. I set correct packages names